src/distribution/uniform.rs

```diff
@@ -464,4 +464,23 @@ mod test {
         test::check_continuous_distribution(&try_create(0.0, 10.0), 0.0, 10.0);
         test::check_continuous_distribution(&try_create(-2.0, 15.0), -2.0, 15.0);
     }
+
+    #[test]
+    fn test_in_range() {
```

Comment:
If we could re-name this to something like `test_sample_in_range` to be a little more explicit that would be awesome